schema/release-schema.json

```diff
@@ -25,6 +25,46 @@
       "format": "date-time",
       "omitWhenMerged": true
     },
+    "publisher": {
+      "title": "Publisher",
+      "description": "Information to uniquely identify the publisher of this specific release. This field is ought to be populated when a release package could contain releases from different publishers.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "title": "Name",
+          "description": "The name of the organization or department responsible for publishing this OCDS release.",
+          "type": "string"
+        },
+        "scheme": {
+          "title": "Scheme",
+          "description": "The scheme that holds the unique identifiers used to identify the publisher being identified.",
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "uid": {
+          "title": "uid",
+          "description": "The unique ID for this publisher under the given ID scheme.",
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "uri": {
+          "title": "URI",
+          "description": "A URI to identify the publisher.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "format": "uri"
+        }
```

Comment:
I'm not sure whether it's better here to align with the publisher object in the package schema, or to use the `Identifier` object from the release schema.

Reply:
I'm not sure either, maybe reusing the Identifier object is the best approach indeed. @jpmckinney what do you think?

Reply:
The Identifier object (which is really an OrganizationIdentifier), would be more consistent.

This will be inconsistent with the release publisher extension, but not many publishers are using it (it is an extension anyway): https://extensions.open-contracting.org/en/extensions/releasePublisher/master/schema/

Please create an issue on the extension registry to decide whether to de-register that extension after 1.2, or add a warning that it is incompatible with 1.2 (which we might have to do for others anyway).

Reply:
Changed to `Identifier` and I created the issue https://github.com/open-contracting/extension_registry/issues/135